test/scripts/start-services.sh

```diff
@@ -4,14 +4,16 @@ set -eu
 # config directory
 configdir=/tmp/snowbridge-e2e-config
 rm -rf $configdir
-mkdir $configdir
+mkdir -p $configdir
 
 # kill all potentially old processes
-kill $(ps -aux | grep -e polkadot/target -e ganache-cli -e release/artemis | awk '{print $2}') || true
+kill $(ps -aux | grep -e polkadot/target -e ganache-cli -e release/artemis | grep -v grep |awk '{print $2}') || true
 
 start_ganache()
 {
     echo "Starting Ganache"
+    pushd ../ethereum
```

Comment:
@yrong It shouldn't be necessary to do this. `ganache-cli` is already installed under the `test` directory if you run `yarn install` in that dir. Are you doing that?

Reply:
ok, gotcha. but i would still suggest to start ganache from ethereum